relay-node/service/src/chain_spec.rs

```diff
@@ -250,10 +264,11 @@ pub fn testnet_config() -> Result<ChainSpec, String> {
         WASM_BINARY.ok_or_else(|| "Development wasm not available".to_string())?,
         Default::default(),
     )
-    .with_name("ZKV Testnet")
+    .with_name("zkVerify Testnet")
     .with_id("zkv_testnet")
-    .with_protocol_id("tacme")
+    .with_protocol_id("tzkv")
     .with_chain_type(ChainType::Live)
+    /*
```

Comment:
why you commented this ?

Reply:
To be able to generate a test chain spec to check the changes in this PR (waiting for the network keys of the regenesis bootnodes); but ok, I can remove the comment in the meantime